aea/cli/add.py

```diff
@@ -48,16 +48,18 @@ def add(ctx: Context, registry):
 
 def _find_connection_locally(ctx, connection_public_id, click_context):
     # check that the provided path points to a proper connection directory -> look for connection.yaml file.
-    # first check in aea dir
+    # first check in registry
     connection_name = connection_public_id.name
     registry_path = ctx.agent_config.registry_path
-    connection_configuration_filepath = Path(os.path.join(registry_path, "connections", connection_name, DEFAULT_CONNECTION_CONFIG_FILE))
+    package_path = Path(registry_path, connection_public_id.author, "connections", connection_name)
```

Comment:
can we make `connections` a constant? Similar for the other pluralised package types.

Reply:
yes good idea, will do